package.nuspec

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+    <metadata>
+        <id>Microsoft.FluentUI.ReactNative</id>
+        <version>$buildNumber$</version>
+        <description>A react-native component library that implements the Fluent Design System.</description>
+        <authors>Microsoft</authors>
+        <projectUrl>https://github.com/microsoft/fluentui-react-native.git</projectUrl>
+        <repository type="git" url="$repoUri$" commit="$commitId$"/>
+    </metadata>
+    <files>
+        <file src="packages/**/*.*" />
+        <file src="scripts/**/*.*" />
```

Comment:
Out of curiosity, what do you need from scripts?

Reply:
Probably nothing. I realize that the code wasn't bundled beforehand and so the nuget package as is might need scripts to help with bundling. 

When I'm not oof, I'll probably investigate having the pipeline bundle / build native apple libraries and pack all of those. That would also mean the package would become apple specific. 😅. Another day though